client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -99,6 +104,12 @@ function isValidCode( serviceName = '', content = '' ) {
 	return includes( content, serviceIds[ serviceName ] );
 }
 
+function jetpackVersionUnsupported( site ) {
```

Comment:
It seems we shouldn't have these kind of functions on the component. What about a selector?